mteb/tasks/Classification/ces/CSFDCZMovieReviewSentimentClassification.py

```diff
@@ -49,3 +50,47 @@ def dataset_transform(self):
         self.dataset = self.stratified_subsampling(
             self.dataset, seed=self.seed, splits=["test"], n_samples=N_SAMPLES
         )
+
+
+class CSFDCZMovieReviewSentimentClassificationV2(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="CSFDCZMovieReviewSentimentClassification.v2",
+        description="""The dataset contains about 30k user reviews from csfd.cz in Czech.
+        This version corrects errors found in the original data. For details, see https://github.com/embeddings-benchmark/mteb/pull/2680""",
```

Comment:
```suggestion
        This version corrects errors found in the original data. These fixes could include removing duplicates, preventing train/test leakage, and more. details, see [the pull request](https://github.com/embeddings-benchmark/mteb/pull/2680) that introduced the fixes.""",
```

I would prefer this description